examples/examples_test.go

```diff
@@ -0,0 +1,129 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package examples
+
+import (
+	"io"
+	"os"
+	"path"
+	"path/filepath"
+	"testing"
+
+	"sigs.k8s.io/service-apis/api/v1alpha1"
+
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/client-go/kubernetes/scheme"
+)
+
+func findExamples(t *testing.T, dirPath string) []string {
+	filenames := []string{}
```

Comment:
[minor] the standard go-style is to do 

`var filenames []string `

for an empty slice (nil is empty)

Reply:
fixed